tests/ast/test_ast_driver.py

```diff
@@ -248,7 +249,8 @@ def run_test(session):
         sys.modules[test_name] = test_module
         spec.loader.exec_module(test_module)
         base64_batches = test_module.run_test(session)
-        unparser_output = render(base64_batches) if pytest.unparser_jar else ""
+        raw_unparser_output = render(base64_batches) if pytest.unparser_jar else ""
```

Comment:
Should we enumerate them?
I.e., First encountered temp table -> SNOWPARK_TEMP_TABLE_1 and so on.
This would allow to keep subsequent API accesses testing for the right thing when two tables are involved e.g.

Reply:
(side note: When comparing protobufs - I removed the temp tables altogether)

Reply:
can we add a comment on why we sub the temp tables here?

Reply:
We're coming across these in the Spark project. This was meant to be a quick change – it's not worth spending the time on the fancier version you're suggesting at the moment.